setup/security/authorization/pipeline-permissions/index.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: single
+title:  "Pipeline Permissions"
+sidebar:
+  nav: setup
+---
+
+{% include alpha version="1.9" %}
+
+{% include toc %}
+
+Pipeline permissions enable automatically triggered pipelines to modify
+resources in protected accounts and applications. They are an alternative
+to manually managing Fiat Service Accounts.
+
+Previously, a Spinnaker operator would have to first create a Fiat Service
+account with the correct permissions. A user could then specify the service
+account as the `RunAsUser` per automated trigger. With Pipeline permissions,
+the flow is simpler -- a user only has to specify the set of roles in the
+pipeline specification. You can read more about how pipeline permissions
+works [here](https://github.com/spinnaker/spinnaker/issues/3037).
```

Comment:
I think we should put all the relevant information here in the doc, rather than linking out to the issue.

Reply:
Removed the reference and added a one line description of how it works.

Reply:
:+1: